sql/src/main/java/io/crate/execution/engine/aggregation/GroupingProjector.java

```diff
@@ -107,6 +112,28 @@ public GroupingProjector(List<? extends Symbol> keys,
         }
     }
 
+    private <T extends Number> GroupBySingleNumberCollector<T> createSingleNumberGroupingCollector(Class<T> inputType,
```

Comment:
Why are we wrapping the constructor in a (non static?) function if there is no additional logic or anything?

Reply:
I initially thought it'd ease the cognitive load on the method calling this, but I suppose it'd be the same as directly calling the constructor